### ISSUE TITLE
2024 Debt Services: Collab Cycle PO Synch

### ISSUE THREAD

**Issue by jennb33** [department-of-veterans-affairs/va-mobile-feature-support#132]
### User Story
As the UX team on the MFS team, 
We need to be involved with the Governance team's Collaboration Cycle for new product development and request a Collab Cycle kickoff request,
So that our development is being done in-line with Governance and VA requirements.

### Overview
When we are in process with the Debt Services Development, we need to follow this guidance by the Governance team to be in sync with the Collaboration Cycle. The PO Synch is the second touchpoint in the Collaboration Cycle.

What’s the purpose of the PO Sync?
- The PO Sync meeting (or “touchpoint”) helps to align OCTO-DE, Governance, and VFS team members before the Collaboration Cycle begins.

This meeting is less a formal presentation and more a discussion. It not only provides OCTO-DE and Platform with an early understanding of the purpose and vision behind your team’s proposed work, but is also an opportunity for OCTO-DE practice area leads to raise any questions or concerns that need OCTO alignment, and ensure that vision and strategy are discussed before work begins to help both product and shared service teams work faster and more confidently together.

Preparing for the PO Sync
When should the PO Sync be scheduled?

Schedule this meeting when:
- You have developed an idea for how to solve your problem statement
- You have drafted a Product Outline
- The PO Sync must be completed before scheduling additional touchpoints in the Collaboration Cycle.
- Note: The OCTO-DE Team will need at least 2 business days to review your materials. Ensure you provide enough lead time, otherwise the meeting may need to be rescheduled.

What artifacts will be needed?
For the PO Sync, only the Product Outline is a required artifact.

The following artifacts should be ready and linked in the VFS team’s Collaboration Cycle Request GitHub ticket at least 2 business days before the scheduled meeting.

Required artifacts
- [Product Outline Template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/product-management/product-outline-template.md)
- Optional artifacts
- Any other relevant artifacts you may want to share

How to schedule the PO Sync
Follow the steps in your team’s Collaboration Cycle Request GitHub ticket.

Schedule: Use the Calendly link in the Before Meeting > VFS Actions section to select a date for the touchpoint meeting. The Governance Team will reach out within a business day via Slack to confirm that your touchpoint has been scheduled.

Add artifacts: Go to the Before meeting > PO Sync artifacts checklist and make sure the requested artifacts are linked.

Please request your PO Sync meeting and provide artifacts at least 2 business days ahead of time so the OCTO-DE Team can review your artifacts.

Who should schedule the PO Sync?
Because the PO Sync is only attended by the OCTO Product Owner and the OCTO-DE practice area leads, the meeting should be scheduled by the OCTO Product Owner.

Who should attend the PO Sync?
Only the OCTO Product Owner and relevant OCTO-DE practice area leads will attend the PO Sync. Other VFS Contractors and Governance Team members should not be invited except in unique circumstances. It is possible that not all OCTO-DE practice area leads will attend, depending on the needs of the initiative.

What happens during the PO Sync?
Meeting format
- 30-minute synchronous meeting on Microsoft Teams
- Hosted by the OCTO-DE practice area leads
- Recorded for later reference

Agenda
Because the information necessary for a successful PO Sync will change from project to project, the meeting agenda will be determined by the OCTO-DE practice area leads. In some cases, relevant Governance Team members will help define the agenda.

What sort of feedback can teams expect?
Teams will receive feedback such as, but not limited to: historical context of similar initiatives (have we tried this before and how did it go?), organizational context (are other teams working on similar initiatives?), and scope as it relates to other teams' workloads.

After the PO Sync Meeting
After the meeting, the OCTO-DE team will add the link to the recording of the meeting to the Collaboration Cycle Request GitHub ticket.

The OCTO-DE practice area leads will determine whether or not the initiative may proceed.

If the initiative receives OCTO alignment to proceed, one of the OCTO-DE practice area leads will add the PO-Sync-approved label to the Collaboration Cycle Request ticket.

If the initiative does not receive OCTO alignment to proceed, one of the OCTO-DE practice area leads will comment on the Collaboration Cycle Request ticket communicating that the initiative will not move forward. The OCTO-DE team will then close the ticket.

If the initiative receives OCTO alignment to proceed, you can then move on to the next touchpoint.

## Tasks
- [ ] Hold PO Sync
- [ ] Document and record meeting
- [ ] Receive feedback or approval to proceed _or_ the initiative will not move forward, so it is closed. 

## Acceptance Criteria
- [ ] PO Sync held and notes are taken during meeting
- [ ] We have approval or have been denied moving forward based on the PO feedback

## Potential Dependencies
- Turn-around time from OCTO PO team to review (2 business days are the standard)

## Out of Scope
- Implementing new product is out of scope for this ticket, this ticket is just to hold the Debt Services PO review meeting
